org.opent2t.sample.binaryswitch.superpopular/com.contosothings.binaryswitch/js/thingTranslator.js

```diff
@@ -93,6 +93,7 @@ class Translator {
                 translator: 'opent2t-translator-com-contosothings-binaryswitch',
                 controlId: this.deviceId
             },
+            availability: 'online',
```

Comment:
Added in "All Platforms" unit test.

---
In reply to: [105797455](https://github.com/openT2T/translators/pull/197#discussion_r105797455) [](ancestors = 105797455)